susemanager-utils/testing/automation/backend-unittest-pgsql.sh

```diff
@@ -1,7 +1,25 @@
 #! /bin/sh
+if [ -z ${PRODUCT+x} ];then
+    PRODUCT="Uyuni"
+fi
+
+while getopts 'P:h' c
+do
+    case $c in
+        P) PRODUCT=$OPTARG ;;
```

Comment:
```suggestion
        P) VPRODUCT="VERSION.${OPTARG}";;
```

If we ever change the filenames, this makes things easier. No need to change a lot of places in the script.